### PR TITLE
Battle Map and Move Disassembly

### PR DESCRIPTION
**Pull request by JoshDuMan**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀 This PR matches 4 functions (+0.02%) on `us`.

✅ This PR fixes 6 warnings!

File: ver/us/splat.yaml [hunk 1]
```diff
@@ -2209,11 +2209,11 @@ segments:
     - [0x325160]
   - type: code
     start: 0x3251D0
-    vram: 0x802BD100
+    vram: 0x802C0000
     overlay: True
     subsegments:
     - [0x3251D0, c]
```
Comment (ethteck) [inline ver/us/splat.yaml:2215]: If you add `, 3251D0` after the `c` here, you can make the next line just `- [0x3255E0]`

**Review by ethteck — APPROVED** [review]

File: include/enums.h [hunk 2]
```diff
@@ -3121,4 +3126,30 @@ enum DictionaryIndex {
     DICTIONARY_SIZE,
 };
 
+enum ActionCommandDmaTable {
```
Comment (nanaian) [inline include/enums.h:3129]:
```suggestion
enum ActionCommand {
```

File: tools/disasm_script.py [hunk 3]
```diff
@@ -149,7 +149,7 @@ def get_constants():
     global VALID_SAVE_VARS
 
     valid_enums = { "StoryProgress", "ItemIDs", "PlayerAnims", "ActorType",
-        "ActorIDs", "Events", "SoundIDs", "SongIDs", "Locations",
+        "ActorIDs", "Events", "SoundIDs", "SongIDs", "Locations",  "ActionCommandDmaTable",
```
Comment (nanaian) [inline tools/disasm_script.py:152]: & here

**Pull request merged**